gratipay/billing/exchanges.py

```diff
@@ -179,6 +179,7 @@ def create_card_hold(db, participant, amount):
     # ===============
 
     cents, amount_str, charge_amount, fee = _prep_hit(amount)
+    amount = charge_amount - fee
```

Comment:
Are you sure this is the right way to do this? On my reading this variable only gets used in the exception case below. How do we call `record_an_exchange` in the success case? What is it expecting for amount? Doesn't this pass different values in the success and error cases?